### DISCUSSION TITLE
504 Gateway Timeout without code changes

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [vercel/next.js#58983] [Help]
### Summary

hi,

For the last 2 days https://journeai.com/api/chat times out with 

504 Gateway Timeout when calling openai. Works locally, i tried increasing the time out.

No success. I rolled back and dont think its a code change

**Comment by 3a1b2c3** [comment]
working now

**Reply by kirtirajsinh** [reply]
What did you change. i am facing the same errror
